workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/views/matplotlib.py

```diff
@@ -89,7 +102,71 @@ def plot_2d_matplotlib(
         template.render(
             image_data=data,
             runs=sorted(run_diags.runs),
-            variables_to_plot=variables_to_plot,
+            variables_to_plot=sorted(variables_to_plot),
+            varfilter=varfilter,
+        )
+    )
+
+
+def plot_cube_matplotlib(
+    run_diags: RunDiagnostics,
+    run_metrics: pd.DataFrame,
+    varfilter: str,
+    metric_names: Mapping[str, str] = None,
+    **opts,
+) -> str:
+    """Plot horizontal maps of cubed-sphere data for all diagnostics whose name includes
```

Comment:
Please add some white space to this docstring. The first line should be short and less than 88 characters.